src/main.ts

```diff
@@ -30,14 +30,19 @@ export const cspClient = clients.newClient(
   }
 )
 
+const defaultHeadersVibClient = {
+  "Content-Type": "application/json",
+  "User-Agent": `vib-action/${userAgentVersion}`,
+}
+
 export const vibClient = clients.newClient(
   {
     baseURL: `${process.env.VIB_PUBLIC_URL ? process.env.VIB_PUBLIC_URL : constants.DEFAULT_VIB_PUBLIC_URL}`,
     timeout: getNumberInput("http-timeout"),
-    headers: {
-      "Content-Type": "application/json",
-      "User-Agent": `vib-action/${userAgentVersion}`,
-    },
+    headers:
+      process.env.VERIFICATION_MODE === "true"
+        ? { ...defaultHeadersVibClient, "x-verification-mode": "SERIAL" }
```

Comment:
Although HTTP headers are case insensitive, it's common to case them, i.e. suggesting to replace with `X-Verification-Mode`

Reply:
A job appears `properties=[Property{headers=x-verification-mode=SERIAL}]}`

![image](https://user-images.githubusercontent.com/17168510/188902178-52c4051f-c220-4f5e-b934-867314e2e582.png)

What is the correct value?

Reply:
I mean doing this change:
```suggestion
        ? { ...defaultHeadersVibClient, "X-Verification-Mode": "SERIAL" }
```